tests/test_ble_interface_core.py

```
@@ -4620,6 +4769,95 @@ def should_not_create_thread(*_args: object, **_kwargs: object) -> None:
+        ident=None,
+        is_alive=lambda: False,
+    )
+    monkeypatch.setattr(
+        iface.thread_coordinator,
+        "_create_thread",
+        lambda **_kwargs: thread_like,
+        raising=True,
+    )
+
+    def _raise_start_failure(_thread: object) -> None:
+        raise RuntimeError("start failed")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)